src/kabasuji/controllers/BoardToBullpenMove.java

```diff
@@ -0,0 +1,46 @@
+package kabasuji.controllers;
+import kabasuji.models.Level;
+import kabasuji.models.PlacedPiece;
+
+/**
+ * Move action  to move a piece from the board to the bullpen
+ * 
+ * @author Logan
+ *
+ */
+public class BoardToBullpenMove implements IMove {
+  
+  
+  Level level;
+  
+  PlacedPiece piece;
+  
+  /**
+   * Constructor
+   * @param level the level that is being played
+   * @param piece the piece to move
+   */
+  public BoardToBullpenMove(Level level, PlacedPiece piece) {
```

Comment:
On second thought... This be used by the level builder I imagine. We need to be able to do that on any type of level.